static/src/stylesheets/layout/_header.scss

```diff
@@ -179,3 +147,648 @@ $c-guardian-services-action: #ffffff;
+            transform: rotate(-135deg);
+        }
+    }
+}
+
+.main-menu__icon {
+    position: absolute;
+    left: $gs-gutter / 2;
+    text-align: center;
+    width: $navigation-horizontal-padding - $gs-gutter/2;
+
+    &__svg {
+        fill: $news-support-2;
+        margin-top: -.05em;
+        width: 1em;
+        height: 1em;
+    }
+}
+
+.main-navigation__secondary {
+    color: #ffffff;
+    background: darken($guardian-brand-dark, 4%);
+
+    .main-menu__icon__svg {
+        margin-top: -.24em;
+        width: .8em;
+        height: .8em;
+    }
+}
+
+.navigation-border,
+[data-link-name='nav2 : the guardian app'] {
+    // Creates a seamless line when section has dark background
+    margin-top: -1px;
+    position: relative;
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: $navigation-horizontal-padding;
+        right: 0;
+        height: 1px;
+        background: darken($guardian-brand-dark, 4%);
+    }
+}
+
+[data-link-name='nav2 : the guardian app'],
+[data-link-name='nav2 : facebook'] {
+    margin-top: $gs-baseline * 2;
+}
+
+.navigation-group {
+    // Unset ul from _lists.scss
+    margin-left: 0;
+    // Unset ul from user agent stylesheet
+    margin-top: 0;
+    // Unset ul from user agent stylesheet
+    margin-bottom: 0;
+}
+
+.navigation-group--contrast {
+    color: $guardian-brand-dark;
+    background-color: $news-main-2;
+    padding-bottom: $gs-baseline;
+
+    .main-menu__icon__svg {
+        fill: $guardian-brand;
+    }
+
+    .personalisation__icon:before {
+        border-color: $guardian-brand;
+    }
+}
+
+.navigation-group__item {
+    // Override inherited ul from user agent stylesheet
+    list-style: none;
+    position: relative;
+    font-size: 1em;
+
+    details[open] &:last-child {
+        padding-bottom: $gs-baseline;
+    }
+
+    a {
+        display: block;
+        padding-top: $gs-baseline / 1.5;
+        padding-right: $gs-gutter;
+        padding-bottom: $gs-baseline / 1.5;
+        padding-left: $navigation-horizontal-padding;
+
+        // Override a from user agent stylesheet
+        color: inherit;
+
+        // TODO: Share with large nav?
+        &:focus,
+        &:hover {
+            text-decoration: none;
+            // Unset a:focus from user agent stylesheet
+            outline: 0;
+        }
+    }
+}
+
+.secondary-navigation {
+    font-size: .9em;
+
+    &:last-child {
+        margin-bottom: $gs-baseline * 3;
+    }
+}
+
+.main-menu__icon--social__svg {
+    margin-top: -.4em;
+    width: 1.8em;
+    height: 1.8em;
+}
+
+// Search bar
+.navigation-group__search-container {
+    font-size: .8em;
+    margin: $gs-baseline / 2 $gs-gutter * 2 $gs-baseline $gs-gutter / 2;
+    overflow: hidden;
+}
+
+.navigation-group__search-box {
+    border: 0;
+    vertical-align: middle;
+    background-color: $guardian-brand;
+    padding-left: $navigation-horizontal-padding - $gs-gutter / 2;
+    border-radius: 1000px;
```

Comment:
Its a good question! Border radius with px values behave differently than % values. To demonstrate:

no border radius:
![image](https://cloud.githubusercontent.com/assets/8774970/24652677/a0dddcbe-192a-11e7-9afe-ff0d61c35a5e.png)

1000px:
![image](https://cloud.githubusercontent.com/assets/8774970/24652665/98e6a770-192a-11e7-87f7-c602d8444351.png)

50%:
![image](https://cloud.githubusercontent.com/assets/8774970/24652690/abbd9962-192a-11e7-9e07-f8e8fa39c2a1.png)

According to https://developer.mozilla.org/en/docs/Web/CSS/border-radius its because a px value rounded the corner to the arc of a circle, whereas a % on a rectangle, will round the corner in the arc of a ellipsis